README.md

```diff
@@ -79,45 +86,14 @@ Install the following command line tools:
 
   ```
 
-* `kubectl crossplane extension` (optional)
-
-  ```console
-  curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.sh | sh
-  cp kubectl-crossplane /usr/local/bin
-  ```
-
 ## Quick Start
 
 ### Platform Ops/SRE: Run your own internal cloud platform
```

Comment:
This heading seems out of place in the overall content hierarchy now with the removal of the install instructions?